.github/workflows/test.yaml

```diff
@@ -80,7 +79,7 @@ jobs:
           --show-error
           --request DELETE
           --header "Accept: application/vnd.github.v3+json"
-          --header "Authorization: token ${{ secrets.GITHUB_TOKEN }}"
+          --header "Authorization: token ${{ github.token }}"
```

Comment:
I would mention the rationale for this change in the commit message since otherwise it seems random.